content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -266,6 +275,16 @@ an `ingressClassName` specified. You can resolve this by ensuring that at most 1
 IngressClass is marked as default in your cluster.
 {{< /caution >}}
 
+There are some ingress controllers, that work without the definition of a default
+`IngressClass`, even on newer versions of Kubernetes and the controllers. For
+example, the Ingress-NGINX controller can be configured with a
+[flag](https://kubernetes.github.io/ingress-nginx/#what-is-the-flag-watch-ingress-without-class)
+`--watch-ingress-without-class`. Although this works, it is highly
+[recommended](https://kubernetes.github.io/ingress-nginx/#i-have-only-one-instance-of-the-ingresss-nginx-controller-in-my-cluster-what-should-i-do)
+that a default `IngressClass` is created:
```

Comment:
Based on https://github.com/kubernetes/website/pull/30737#discussion_r763158014, could you make sure that this sentence is also modified? I think it would be a lot cleaner if it matches the one above

Reply:
done!